dist/bin/kctf-cluster

```diff
@@ -747,12 +836,14 @@ function kctf_cluster_resize {
 function kctf_cluster_usage {
   echo -e "usage: kctf cluster command" >&2
   echo -e "available commands:" >&2
-  echo -e "  create: create a new cluster config" >&2
-  echo -e "  list:   list available cluster configs" >&2
-  echo -e "  load:   load an existing cluster config" >&2
-  echo -e "  start:  start the cluster" >&2
-  echo -e "  stop:   stop the cluster" >&2
-  echo -e "  resize: resize the cluster" >&2
+  echo -e "  create:            create a new cluster config" >&2
+  echo -e "  list:              list available cluster configs" >&2
+  echo -e "  load:              load an existing cluster config" >&2
+  echo -e "  start:             start the cluster" >&2
+  echo -e "  stop:              stop the cluster" >&2
+  echo -e "  resize:            resize the cluster" >&2
+  echo -e "  set-src-ip-ranges: allow a range of IPs to access the services" >&2
+  echo -e "  get-src-ip-ranges: print the current allowed IP range" >&2
```

Comment:
nit: plural in name singular in text. it's ranges I guess

Reply:
done